web-bundle/src/main/java/com/graphhopper/replica/ReplicaCustomSpeedsBikeTagParser.java

```diff
@@ -9,25 +9,40 @@
 import com.graphhopper.routing.ev.EdgeIntAccess;
 import com.graphhopper.routing.ev.EncodedValueLookup;
 import com.graphhopper.routing.util.parsers.BikeAverageSpeedParser;
+import com.graphhopper.stableid.StableEdgeIdManager;
 import com.graphhopper.util.PMap;
-import org.apache.commons.lang3.NotImplementedException;
 
 import static com.graphhopper.customspeeds.CustomSpeedsUtils.*;
 
 public class ReplicaCustomSpeedsBikeTagParser extends BikeAverageSpeedParser {
     private final ImmutableMap<CustomSpeedId<?>, CustomSpeed<?>> idToCustomSpeeds;
+    private final StableEdgeIdManager stableEdgeIdManager;
 
     public ReplicaCustomSpeedsBikeTagParser(EncodedValueLookup lookup, PMap configuration,
-                                            ImmutableMap<CustomSpeedId<?>, CustomSpeed<?>> idToCustomSpeeds) {
+                                            ImmutableMap<CustomSpeedId<?>, CustomSpeed<?>> idToCustomSpeeds,
+                                            StableEdgeIdManager stableEdgeIdManager) {
         super(lookup, configuration);
         this.idToCustomSpeeds = idToCustomSpeeds;
+        this.stableEdgeIdManager = stableEdgeIdManager;
     }
 
     @Override
     public void handleWayTags(int edgeId, EdgeIntAccess edgeIntAccess, ReaderWay way) {
         super.handleWayTags(edgeId, edgeIntAccess, way);
         if (getCustomSpeedIdType(idToCustomSpeeds) == CustomSpeedId.IdType.SEID) {
-            throw new NotImplementedException("SEID-based custom speeds are not yet implemented for bikes!");
+            ImmutableMap<String, Double> stableEdgeIdToMaxSpeed = getStableEdgeIdBasedCustomSpeeds(idToCustomSpeeds);
```

Comment:
Same note here as in `TruckAverageSpeedParser`; a non-custom speed is assumed to have been set for this edge before the new logic is called